src/Dockerfile.template

```diff
@@ -2,21 +2,19 @@ FROM python:3.8-slim
 
 RUN pip install --upgrade pip
 
-# If requirements.txt exist, copy it and install the packages to enable docker caching layer.
-# Note: the assumption is requirements.txt don't have a relative local package
-COPY *requirements.txt requirements.txt
-RUN if [ -f "requirements.txt" ]; then \
-        pip install -r requirements.txt; \
+# This is to enable Docker caching - if setup.py exists, we install the dependencies before
+#   copying all other files
+COPY *setup.py /setup.py
```

Comment:
do we not copy over the requirements.txt anymore?

Reply:
we will copy it over as part of copying all files and install it in [1]